### ISSUE TITLE
Val loss does not decrease

### ISSUE THREAD

**Issue by Ulrich-1995** [kyotovision-public/dynamic-3d-gaze-from-afar#10]
Hi, there.
I have tried training your model on GAFA dataset from scratch, and there seems to be an overfitting.
The direction_loss and kappa_loss continue to decrease on the training dataset, however, the val_loss and val_mae hit mininum at the end of first epoch, and then keep increasing.
The learning rates for both direction_loss and kappa_loss start at 1e-4.
How did you avoid the overfit when you trained your model?
Thank you so much in advance
